adafruit_hue.py

```diff
@@ -51,11 +61,11 @@ def __init__(self, wifi_manager, bridge_ip=None, username=None):
         self._ip = bridge_ip
         self._username = username
         if bridge_ip and username is not None:
-            self._bridge_url = "http://{}/api".format(self._ip)
+            self._bridge_url = f"http://{self._ip}/api"
             self._username_url = self._bridge_url + "/" + self._username
 
     @staticmethod
-    def rgb_to_hsb(rgb):
+    def rgb_to_hsb(rgb: tuple) -> tuple:
```

Comment:
Unfortunately, the ability to use builtins as generics isn't available in Python 3.7, which is the lowest version supported by Blinka.  So you'd have to use `Tuple` as imported from `typing` for the way you've typed it.  BUT...

Additionally, since it's a generic, we can add more information to the typing.  Since this can be a `tuple` or `list`, I would say the best way to type this would be `Sequence[int]`, which allows both types.  Make sure to import `Sequence` from `typing`!

Reply:
I assumed both tuples?